docs/topics/multiplatform/multiplatform-configure-compilations.md

```diff
@@ -371,4 +371,29 @@ Language settings are checked for consistency in the following ways:
 * `jvmMain` should enable all unstable language features that `commonMain` enables (there's no such requirement for 
 bugfix features).
 * `jvmMain` should use all experimental annotations that `commonMain` uses.
-* `apiVersion`, bugfix language features, and `progressiveMode` can be set arbitrarily.
\ No newline at end of file
+* `apiVersion`, bugfix language features, and `progressiveMode` can be set arbitrarily.
+
+## Configure Isolated Projects feature in Gradle
+
+> This feature is [Experimental](components-stability.md#stability-levels-explained) and is currently in a pre-alpha state with Gradle. 
+> Use it only with Gradle versions 8.10 or higher, and solely for evaluation purposes. The feature may be dropped or changed at any time.
+> We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-57279/Support-Gradle-Project-Isolation-Feature-for-Kotlin-Multiplatform). 
+> Opt-in is required (see details below).
+> 
+{ style="warning" }
+
+Gradle provides the [Isolated Projects](https://docs.gradle.org/current/userguide/isolated_projects.html) feature,
+which improves build performance by "isolating" individual projects from each other. The feature separates build scripts
+and plugins, allowing projects to run safely in parallel.
```

Comment:
> The feature separates build scripts
and plugins, allowing projects to run safely in parallel.

it is not entirely correct. I see you've tried to re-word official explanation by Gradle.

> This means that build logic, such as build scripts or plugins, applied to a project cannot directly access the mutable state of another project

I.e. build scripts and plugins are not separated. But the different projects where they is applied.

Reply:
Is this better?

> The feature separates the build scripts and plugins between projects, allowing them to run safely in parallel.

Reply:
yes it is good enough for the Kotlin documentation. for more accurate and detailed explanation readers can always visit official Gradle documentation.